File: albumloader/src/main/java/com/lee/album/entity/AlbumInfoEntity.java
package com.lee.album.entity;

import java.io.Serializable;
import java.util.List;

/**
 * Author : 李嘉伦
 * e-mail : lijialun@angogo.cn
 * date   : 2021/2/2410:49
 * Package: com.shyz.picture.entity
 * desc   : 相册分类
 */
public class AlbumInfoEntity implements Serializable {
    private List<GalleryInfoEntity> galleryInfo;
    private String albumName;
    private String dirPath;
    private boolean isAll;
    private Integer sort;
    private boolean isSelected;

    public boolean isSelected() {
        return isSelected;
    }

    public void setSelected(boolean selected) {
        isSelected = selected;
    }

    public Integer getSort() {
        return sort;
    }

    public void setSort(Integer sort) {
        this.sort = sort;
    }

    public boolean isAll() {
        return isAll;
    }

    public AlbumInfoEntity setAll(boolean all) {
        isAll = all;
        return this;
    }

    public List<GalleryInfoEntity> getGalleryInfo() {
        return galleryInfo;
    }

    public AlbumInfoEntity setGalleryInfo(List<GalleryInfoEntity> galleryInfo) {
        this.galleryInfo = galleryInfo;
        return this;
    }


    public String getAlbumName() {
        return albumName;
    }


    public String getDirPath() {
        return dirPath;
    }

    public AlbumInfoEntity setDirPath(String dirPath) {
        this.dirPath = dirPath;
        return this;
    }

    private String currentName;

    public String getCurrentName() {
        return currentName;
    }

    public void setCurrentName(String currentName) {
        this.currentName = currentName;
    }

    public AlbumInfoEntity setAlbumName(String name) {
        this.albumName = name;
        return this;
    }


}

File: albumloader/src/main/java/com/lee/album/entity/AlbumInfoEntity.kt
package com.lee.album.entity

import com.lee.album.entity.GalleryInfoEntity
import com.lee.album.entity.AlbumInfoEntity
import java.io.Serializable

/**
 * Author : 李嘉伦
 * e-mail : lijialun@angogo.cn
 * date   : 2021/2/2410:49
 * Package: com.shyz.picture.entity
 * desc   : 相册分类
 */
class AlbumInfoEntity : Serializable {
    var galleryInfo: MutableList<GalleryInfoEntity>? = null
        private set
    var albumName: String? = null
        private set
    var dirPath: String? = null
        private set
    var isAll = false
        private set
    var sort: Int? = null
    var isSelected = false
    fun setAll(all: Boolean): AlbumInfoEntity {
        isAll = all
        return this
    }

    fun setGalleryInfo(galleryInfo: MutableList<GalleryInfoEntity>?): AlbumInfoEntity {
        this.galleryInfo = galleryInfo
        return this
    }

    fun setDirPath(dirPath: String?): AlbumInfoEntity {
        this.dirPath = dirPath
        return this
    }

    var currentName: String? = null
    fun setAlbumName(name: String?): AlbumInfoEntity {
        albumName = name
        return this
    }
}